server/api/db/sqldb/db.py

```diff
@@ -2577,7 +2560,8 @@ def _get_project_record(
         return project_record
 
     def verify_project_has_no_related_resources(self, session: Session, name: str):
-        artifacts = self._find_artifacts(session, name, "*")
+        artifact_records = self._find_artifacts(session, name, "*")
+        artifacts = [artifact for artifact, _ in artifact_records]
```

Comment:
Since you do this many times, maybe it should be a facility of find_artifacts?
`attatch_tags=True/False` or something